src/config.rs

```diff
@@ -569,6 +571,9 @@ impl Config {
       None
     };
 
+    let unstable = matches.is_present(arg::UNSTABLE)
+      || std::env::var_os(UNSTABLE_KEY).map_or(false, |val| val.to_string_lossy() == "true");
```

Comment:
I'd be inclined to inline this:
```suggestion
      || std::env::var_os("JUST_UNSTABLE").map_or(false, |val| val.to_string_lossy() == "true");
```